core/src/main/resources/messages/TodosaurusCore.properties

```diff
@@ -5,6 +5,8 @@
 action.CreateNewIssue.text=Create New Issue
 action.OpenReportedIssueInBrowser.text=Open Issue In Browser
 action.OpenNewIssueInBrowser.text=Open in Browser
+action.ForgetChoiceAndTryAgainAction.text=Forget Choice And Try Again
+action.ForgetChoice.text=Forget Choice
```

Comment:
_(Minor)_ I am starting to think that "Forget choice" is a confusing name for an action.

Imagine that you are reporting an issue, see an error, and is suddenly presented by an option to "forget choice", while you didn't do any choice in the current IDE session, and, ironically, might even forget about doing it previously.

Let's maybe name this "Choose another account", "Choose another tracker", or something alike?

Reply:
I agree. I'll fix it.

Reply:
So far I have replaced the text in these actions. 
Do I need to replace “Forget choice” with the new wording in Wizard? 
![image](https://github.com/user-attachments/assets/a7889cd0-7546-4bf2-bc63-3f2326d8a39d)

And in the settings?
![image](https://github.com/user-attachments/assets/ea328a5a-dc04-4ab3-86df-db172f0b7f13)

Reply:
I'd say yes for both cases. Let's have it "Clear saved data" in settings, and "Choose another tracker" (since we restart from the tracker step) in the wizard.